### DISCUSSION TITLE
We can customize the logs viewer view page ?

### DISCUSSION THREAD

**Discussion by AmraniCh** [opcodesio/log-viewer#295] [Q&A]
Hey, thanks for the package! I want to know if it possible to modify the logs viewer interface ?

**Comment by arukompas** [comment]
Hey @AmraniCh unfortunately it's not :) making it extensible would complicate the project for both maintainers and users.

There are no near plans to make that happen either. That is just not the focus of it.

If you'd like to customise the data columns displayed for the logs, that's possible by extending a log type and changing its column definitions. See here - https://log-viewer.opcodes.io/docs/3.x/log-types/default

Hope this helps :)

**Comment by AmraniCh** [comment]
> making it extensible would complicate the project for both maintainers and users.

Emm, actually in my opinion it would be not complicated as long as this feature would be optional not required, some other laravel logs viewer libraries like [rap2hpoutre/laravel-log-viewer](https://github.com/rap2hpoutre/laravel-log-viewer) have this feature with no complicating steps to get it works and customize the view, I'm talking here about the general idea I'm not telling you this is an easy feature to implement because I don't know how this library works behind the scenes.

And thanks for letting me know about logs types feature.
